apps/accounts/forms.py

```diff
@@ -776,3 +776,23 @@ class PreviewForm(forms.Form):
     """
 
     pass
+
+
+class LinkedDomainFormStep0(forms.ModelForm):
+
+    primary = forms.TypedChoiceField(
```

Comment:
if this is a boolean, would you mind using `is_primary` instead of `primary`? I've tried to loosely follow the `is_something` convention elsewhere, and it's referenced in the Octopus Energy django coding conventions, which we've looked before to when making coding style decisions

https://github.com/octoenergy/public-conventions/blob/master/conventions/django.md#model-field-naming-conventions